bundles/org.openhab.voice.googletts/README.md

```diff
@@ -50,6 +50,20 @@ It is recommended to clear this configuration parameter afterwards.
 When enabled the cache is purged once.
 Make sure to disable this setting again so the cache is maintained after restarts.
 
+In case you would like to setup the service via a text file, create a new file in `$OPENHAB_ROOT/conf/services` named `googletts.cfg`
+
+It's contents should look similar to:
```

Comment:
Typo: "Its"

Reply:
Zut, it is wrong in several places.

Reply:
Fixed

Reply:
Had to Google that... visiting Paris in April, so thanks. :-)